collections/_article/rendering-priorities-september-2024.md

```diff
@@ -0,0 +1,115 @@
+
+Godot 4 has brought many quality of life features to particles like [animated velocities](https://godotengine.org/article/progress-report-state-of-particles/) and [turbulence](https://github.com/godotengine/godot/pull/55387). However, particles still fall short in term of larger scale workflow. VFX artists have to come up with their own structure for VFX (often script + animation player), figure out ways to check timings of their effects outside the engine (using screen capture software, for example), modify billboarding logic manually when moving to custom particle shaders, and so on. Despite improvements to the flexibility of the engine, artists still need advanced knowledge to do many things (like write post processing effects with the new CompositorEffect API). 
+
+We still have a lot of work to make Godot more accessible for people that are less interested in learning shaders or programming but still want juicy effects and immersive environments in their games.
+
+If you want to help, there are plenty of things to do. From helping by gathering knowledge and evaluating solutions from multiple perspectives, to implementing agreed [new features and bug fixes](https://github.com/orgs/godotengine/projects/54/views/6), finally, by joining us in the [vfx server](https://discord.gg/HX4xAGaGjm).
+
+* **Customized rendering**
+
+As Godot adoption increases, it is being used for more niche and specialized projects. However, we can't provide everything that every user needs out of the box without becoming bloated, slow, and hard to use. So we need to provide more tools to customize rendering for projects that need more specialized effects. We started that process in 4.3 with the [CompositorEffects API](https://docs.godotengine.org/en/latest/classes/class_compositoreffect.html). We want to continue that effort by implementing more things in the Compositor API to allow users to [control the order of rendering](https://github.com/godotengine/godot-proposals/issues/7916), override [shader templates](https://github.com/godotengine/godot-proposals/issues/8366), etc.
+
+* **Significantly improve screen-space effects (SSAO, SSIL, SSR, SSS)**
+
+All of the screen-space effects in Godot 4 were designed to run without using temporal reprojection for 2 reasons: 1) we didn't want to force anyone to enable TAA in order to have a good looking game and 2) we didn't properly calculate motion vectors for moving objects until Godot 4.2. Temporal reprojection allows us to make our effects much higher quality for a lower base performance cost. 
+
+Now that we have proper motion vectors, we want to provide versions of all the screen-space effects that take advantage of motion vectors to increase their quality and reduce performance cost.
+
+We still won't rely on TAA for any of the built in effects as we want users to be able to choose whether to enable TAA or not, but we will likely use temporal reprojection by default for most of these effects.
```

Comment:
"Require" is more accurate here, as the built-in effects would have the *ability* to rely on TAA in the future.

```suggestion
We still won't require TAA for any of the built-in effects as we want users to be able to choose whether to enable TAA or not, but we will likely use temporal reprojection by default for most of these effects.
```